R/meetup_query.R

```diff
@@ -0,0 +1,26 @@
+
+#' General Meetup query
+#'
+#' Create your own query to the Meetup API
```

Comment:
Add a comment about this being useful for endpoints we don't support with high-level helpers yet?

Reply:
sorry, all your comments didnt become visible when I started! So I overran some of them!